### PR TITLE
MGMT-15240:  Remove code coverage from unit test job

### PR DESCRIPTION
**Pull request by nmagnezi**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -81,19 +81,5 @@ generate-mocks:
 unit-test:
 	$(MAKE) _unit_test TIMEOUT=30m TEST="$(or $(TEST),$(shell go list ./...))"
 
-
 _unit_test: $(REPORTS)
-	gotestsum $(GO_UNITTEST_FLAGS) $(TEST) $(GINKGO_UNITTEST_FLAGS) -timeout $(TIMEOUT) || ($(MAKE) _post_unit_test && /bin/false)
```
Comment (danielerez) [inline Makefile:86]: Maybe worth comment out the coverage instead? As we're probably gonna support it later on

Comment (nmagnezi) [reply Makefile:86]: We should add it when relevant. leaving it here might be confusing

Comment (danielerez) [reply Makefile:86]: Confusing who:) we know we need it soon. But ok, we can add a TODO comment for it at least.

Comment (nmagnezi) [reply Makefile:86]: done

**Pull request merged**